collections/_posts/2022-10-03-MomentumResidualNeuralNetwork.md

```diff
@@ -0,0 +1,151 @@
+---
+layout: review
+title: "Momentum Residual Neural Networks"
+tags: Invertible ResNets
+author: "Stéphanie Jehan-Besson"
+cite:
+    authors: "Michael E. Sander, Pierre Ablin, Mathieu Blondel, Gabriel Peyré"
+    title:   "Momentum Residual Neural Networks"
+    venue:   "ICML 2021"
+pdf: "https://arxiv.org/pdf/2102.07870.pdf" 
+---
+
+<!-- Do not include the following disclaimer in your own review -->
+
+# Reminders on Residual Neural Networks
+
+Seminal work on Residual Neural Network : ["Deep Residual Learning for Image Recognition", K. He, X. Zhang, R. Zen and J. Sun, CVPR 2016](https://arxiv.org/pdf/1512.03385.pdf)
+
+Main points:
+
+* Deeper neural networks are more difficult to train
+* Residual learning framework have been introduced to ease the training of deeper networks
+* Layers are reformulated as learning residual functions with reference to the previous layer inputs, instead of learning unreferenced functions.
+* These residual networks are easier to optimize, and can gain accuracy from considerably increased depth.
+
+
+# Highlights of the paper
+
+* Exploration of Memory costs issues in residual neural network due to backpropagation schemes
+* Proposition of a new model, Momentum ResNets, which circumvents these memory issues by being invertible
+* Compared to other invertible models, this model is simple to integrate into the usual ResNets architectures and provides a rigorous mathematical setting.
+* Momentum ResNets can be interpreted as second order ordinary differential equations (ODEs).
+Momentum ResNets separate point clouds that ResNets fail to separate
+
+## Bibliography in the field 
```

Comment:
I'm not sure if this is the appropriate place to put more bibliographic references. Maybe move this section at the end, as suggestions for further reading for people interested in the paper.

Reply:
This is fair. I dropped out the section.